packages/application/src/shell.ts

```diff
@@ -1303,6 +1314,26 @@ export class LabShell extends Widget implements JupyterFrontEnd.IShell {
     return layout;
   }
 
+  /**
+   * Toggle the fullscreen mode based on user inputs
+   */
+  setFullscreenMode(): void {
```

Comment:
Why do we need this method? Could it be handled in the setter of `fullscreenMode` property instead to minimize the API surface?